src/seedu/addressbook/data/person/Person.java

```diff
@@ -87,5 +87,11 @@ public int hashCode() {
     public String toString() {
         return getAsTextShowAll();
     }
-
+    /**
+     * Compares the name lexicographically using {@link Name#compareTo(Name)}.
+     */
+    @Override
+    public int compareTo(Person other) {
+        return this.name.compareTo(other.name);
```

Comment:
Good one! Though you are repeating the implementation of comapreTo in Name class as well, it helps the Person class not to worry about what properties/behaviours required to sort a Name...